dataikuapi/dss/project.py

```diff
@@ -591,6 +591,14 @@ def get_ml_task(self, analysis_id, mltask_id):
         """
         return DSSMLTask(self.client, self.project_key, analysis_id, mltask_id)
 
+    def list_mltask_queues(self):
+        """
+        List all ML task queues in this project
```

Comment:
Which state for the queues? Is there one queue per MLTask even when there is nothing in it and it's not paused?

Reply:
I guess it's the list of ML task queues that were used in some way since the last time DSS was launched

Reply:
Indeed not easy to explain as a concept.

Does it include non-paused, empty queues?

Reply:
If the queue was interacted with at any point while DSS was running, it will be list of queues. Empty queues that were run at some point are still in the list.

Reply:
I guess that it would be less confusing (and befitting the one known use case) to return only the queues that are non empty and/or paused, as well as some additional info. Will comment on the backend API.